Jenkinsfile.github

```diff
@@ -16,8 +16,24 @@ pipeline {
         GIT_REPO_NAME = getRepoName()
         RELEASE_BRANCH_VERSION = getReleaseBranchVersion()
     }
-    
+
     stages {
+        stage("Update Vendored Dependencies") {
+            steps {
+                script {
+                    sh(returnStdout: true, script: "git vendor list").split("\n").findAll{it =~ /^[a-z]/}.each {
+                        it = it.replaceFirst(/:$/, "")
+                        echo "Updating vendor dependency ${it}"
+                        (vendorName, vendorBranch) = it.split('@')
+                        sh "git vendor update ${vendorName} ${vendorBranch}"
+                        // RPM build may get confused on multiple spec files in tarball
+                        echo "Removing spec files from vendored dependencies"
+                        sh "find . -mindepth 2 -name '*.spec' -delete"
```

Comment:
Remove spec files other then docs-csm.spec in root folder. Otherwise, rpmbuild is confused when reading multiple spec files from tarball.